platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/preconsensus/InlinePcesWriter.java

```diff
@@ -43,7 +43,7 @@ public interface InlinePcesWriter {
      * @return the event written
      */
     @InputWireLabel("events to write")
-    @NonNull
+    @Nullable
```

Comment:
Why is it nullable? It should not drop any events

Reply:
reverted